Sources/AppStoreConnectCLI/Commands/Reports/DownloadSalesAndTrendsReportsCommand.swift

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+import FileSystem
+
+struct DownloadSalesAndTrendsReportsCommand: CommonParsableCommand {
+
+    // swiftlint:disable identifier_name
+    enum ReportFilter: String, ExpressibleByArgument, CaseIterable, CustomStringConvertible {
+
+        // SALES
+        case sales_daily_summary
```

Comment:
The enum names don't really match swift style. But I understand why you did it this way.  The other way would be to keep the cases in line with Swift style (eg, camelCase) but then assign snake_case strings as values.

Reply:
Or you could write a utility to convert a camelCase value to a snake_case value and call that in `description` on `rawValue`.